dao/README.md

```diff
@@ -0,0 +1,119 @@
+# Croncat DAO
+Enabling a community to own, grow & maintain the blockchain scheduling utility.
+
+# Mission
+Provide a well balanced group of persons capable of furthering the development of Croncat, maintaining core business objectives that sustain the network & act on community improvement initiatives.
+
+### Core Values
+* Stability
+* Economic Sustainability
+* Community Ownership
+
+# Are you a Croncat?
+Croncat is an organism owned & maintained by dedicated people coming from a diverse set of perspectives. 
+Here are the classifications of what makes up the types of people in Croncat DAO:
+* **Founder** - Core contributors that provide vision, implementation & leadership
+* **Agent** - The operator executing the tasks & runtime dictated by the DAO
+* **Application** - Entities that need scheduled transactions
+* **Commander** - Individuals that contribute to initiatives defined by the DAO, receive retainer stipend based on performance (Example: Growth Hacker)
+
+Becoming part of the Croncat DAO is much different than token based DAOs. It requires real interaction and participation in community development, governance, research & network growth. To maintain a seat on Croncat DAO, you believe in the vision, aspire to further the mission statement, and generally provide your personal perspective to create positive outcomes for the Croncat community as a whole. Maintaining a seat on the DAO requires interactions like voting at minimum 5 times per year.
+
+# Governance & Operations
+### Council Responsibilities
+DAO council will be the sole responsible entity for the development and promotion of croncat. The council will be made up of different types of persons, each bringing a unique viewpoint to the governance process to align and balance the DAO. You can think of the council representing 3 core entities: Founders, Producers, Consumers. The council will have the power to enact proposals, fund development & marketing, provide guidance on integrations and onboard further community members.
+
+Council members are responsible to the DAO, and have a requirement to on-chain activity periodically to maintain their council seat. By staying active on the chain activities, members not only keep the community inline with their perspective, but also help create a wider base for decisions. Council members are also responsible for doing the research and diligence necessary to make sound decisions for fund allocations.
```

Comment:
might expand on "on-chain activity" here in the first sentence. Doing agent stuff? Voting?

Reply:
I will handle that in the docs side, this was just for reference of why the policy was set this way to start.